docs/reference/crds/verifiers.md

```diff
@@ -0,0 +1,57 @@
+Ratify supports many verifiers to validate different artifact types. View more CRD samples [here](../../../config/samples/). Each verifier must specify the name of the verifier and the artifact type this verifier handles. Common properties:
+
+```yml
+name: required, name of the verifier
+artifactType: required, the type of artifact this verifier handles
+address: optional. Plugin path, defaults to value of env "RATIFY_CONFIG" or "~/.ratify/plugins"
+parameters: optional. Parameters specific to this verifier
+```
+ 
+
+## Notary
+
+Sample Notary yaml spec.  :
+```yml
+apiVersion: config.ratify.deislabs.io/v1alpha1
+kind: Verifier
+metadata:
+  name: verifier-notary
+spec:
+  name: notaryv2
+  artifactTypes: application/vnd.cncf.notary.signature
+  parameters:
+    verificationCertStores:  #certificates for validating signatures
+      certs: # name of the trustStore
+          - certstore-akv # name of the certificate store CRD to include in this trustStore
```

Comment:
looks like this line indents 3 spaces.

Reply:
thanks for catching these typos! pushed an update.